package.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
Why did you delete this file? Please revert.

Reply:
Sorry, readded this.